### PR TITLE
style(Menu): set min width of dropdown menu item

### PR DESCRIPTION
**Pull request by TomasSlama**
[FX-3127]

### Description

Showcase how to use Dropdown menu items with descriptions. Also set `min-width` to comply with BASE. I asked Matt directly about this change and he agreed.

### How to test

- Check the [new example](https://picasso.toptal.net/fx-3127-dropdown-with-description/?path=/story/components-dropdown--dropdown#dropdown-menu) in the storybook and compare it with [the design](https://www.figma.com/file/5SCTOPrCDcHuk5We091GBn/Product-Library?node-id=202%3A10632). Be aware that the icon example will be done in the next PR.
- Check Happo screenshots

### Screenshots

image

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3127]: https://toptal-core.atlassian.net/browse/FX-3127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/picasso/src/MenuItem/styles.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ PicassoProvider.override(() => ({
 export default ({ typography, palette, shadows }: Theme) =>
   createStyles({
     root: {
+      minWidth: rem('144'),
```
Comment (TomasSlama) [inline packages/picasso/src/MenuItem/styles.ts:30]: discussed with Matt

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

File: packages/picasso/src/Menu/story/index.jsx [hunk 2]
```diff
@@ -19,10 +19,9 @@ page
   .createChapter()
   .addExample('Menu/story/Default.example.tsx', {
     title: 'Default',
-    takeScreenshot: false,
   })
   .addExample('Menu/story/Nested.example.tsx', {
-    title: 'Nested',
+    title: 'Multilevel',
```
[the remark this answers is not in the document]
Comment (mkrl) [reply packages/picasso/src/Menu/story/index.jsx:24]: Nice! But now you can't find it by searching "nested" in our brand new advanced search :smile:

**Review by mkrl — APPROVED** [review]
:ok_hand:

File: packages/picasso/src/Menu/story/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply packages/picasso/src/Menu/story/index.jsx:24]: Should we also change file name?

**Review by ozgurkececioglu — APPROVED** [review]
good work 👍 didn't test
thank you Tomas 🙇‍♂️

**Review by OndrejTuma — APPROVED** [review]
Tested in temploy, compared to design 👌 
Code looks good

**Review by augustobmoura — APPROVED** [review]
Nice work @TomasSlama! :+1:

**Pull request merged**